CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/compare/CompareGraphAction.java

```diff
@@ -41,8 +45,15 @@ public CompareGraphAction(final GraphNode context) {
 
     @Override
     public void actionPerformed(final ActionEvent e) {
-        PluginExecution.withPlugin(CorePluginRegistry.COMPARE_GRAPH)
+        // Check current graphs opened
+        if(GraphManager.getDefault().getAllGraphs().size() > 1){
```

Comment:
Are you able to add a space between `if` and `(`. Similar for `)` and `{`

Reply:
I'll have to look at my onSave() formatting, usually this is auto formatted out. fixed.